db/data_migration/20160211164516_republish_topical_event_about_pages_to_publishing_api.rb

```diff
@@ -0,0 +1 @@
+AboutPage.where.not(content_id: nil).each(&:publish_to_publishing_api)
```

Comment:
Thanks!
- I was reusing [this](https://github.com/alphagov/whitehall/blob/master/db/data_migration/20160205105837_republish_topical_events_to_publishing_api.rb#L1) and I thought publishing something without a content_id might cause issues with the publishing-api so I left it. Just checked in integration though and none of the topical event about pages have a nil content_id.
- Regarding your second comment: makes sense, amended now!